util/kube/kube.go

```diff
@@ -26,70 +22,6 @@ func SetAppInstanceLabel(target *unstructured.Unstructured, key, val string) err
 	}
 	labels[key] = val
 	target.SetLabels(labels)
-	if key != common.LabelKeyLegacyApplicationName {
```

Comment:
Remove the legacy labeling statement from the function description as well. Also, as there is no chance of getting errors anymore, remove returning of error as part of function definition.

Reply:
Fixed